docs/sources/logql/analyzer/script.js

```diff
@@ -1,4 +1,4 @@
-let host = "https://logql-analyzer.grafana.net/2-6-x";
+let host = "https://logql-analyzer.grafana.net/next";
```

Comment:
is it just me or both `https://logql-analyzer.grafana.net` and `https://logql-analyzer.grafana.net/next` is unavailable?

Reply:
we need to do POST requests to 
* https://logql-analyzer.grafana.net/next/api/logql-analyze
* https://logql-analyzer.grafana.net/2-6-x/api/logql-analyze

I checked it with postman, it works